### PR TITLE
3-D to 3D

### PR DESCRIPTION
**Pull request by gewarren**
According to style guide, it should be written '3D': https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=28784

Also, I lost power/wifi for a couple hours today, so I did a bunch of chore work like this that I could do offline.

### CODE REVIEW

File: docs/framework/wpf/graphics-multimedia/how-to-animate-a-3-d-rotation-using-rotation3danimation.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ The following example shows how to make a 3D object rotate while it "wobbles" by
 ## See also
 
 - [3-D Graphics Overview](3-d-graphics-overview.md)
```
Comment (Thraka) [inline docs/framework/wpf/graphics-multimedia/how-to-animate-a-3-d-rotation-using-rotation3danimation.md]: For example here

File: docs/framework/wpf/graphics-multimedia/how-to-animate-a-3-d-rotation-using-storyboards.md [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
 ## Example  
  [!code-xaml[Animation3DGallery_snip#Rotate3DUsingAxisAngleRotation3DExampleWholePage](~/samples/snippets/csharp/VS_Snippets_Wpf/Animation3DGallery_snip/CS/Rotat3DUsingAxisAngleRotation3DExample.xaml#rotate3dusingaxisanglerotation3dexamplewholepage)]  
   
 ## See also
 
-- [Animate a 3-D Rotation Using Rotation3DAnimation](how-to-animate-a-3-d-rotation-using-rotation3danimation.md)
-- [Animate a 3-D Rotation Using Key Frames (Rotation3DAnimationUsingKeyFrames)](how-to-animate-a-3-d-rotation-using-key-frames.md)
+- [Animate a 3D Rotation Using Rotation3DAnimation](how-to-animate-a-3-d-rotation-using-rotation3danimation.md)
+- [Animate a 3D Rotation Using Key Frames (Rotation3DAnimationUsingKeyFrames)](how-to-animate-a-3-d-rotation-using-key-frames.md)
 - [3-D Graphics Overview](3-d-graphics-overview.md)
```
Comment (Thraka) [inline docs/framework/wpf/graphics-multimedia/how-to-animate-a-3-d-rotation-using-storyboards.md]: And here

File: docs/framework/wpf/graphics-multimedia/how-to-apply-emissive-material-to-a-3-d-object.md [hunk 3]
```diff
@@ -1,15 +1,15 @@
...
-title: "How to: Apply Emissive Material to a 3-D Object"
+title: "How to: Apply Emissive Material to a 3D Object"
 ms.date: "03/30/2017"
 dev_langs: 
   - "csharp"
   - "vb"
 helpviewer_keywords: 
-  - "EmissiveMaterial [WPF], applying to 3-D objects"
+  - "EmissiveMaterial [WPF], applying to 3D objects"
   - "3-D objects [WPF], applying EmissiveMaterial"
```
Comment (Thraka) [inline docs/framework/wpf/graphics-multimedia/how-to-apply-emissive-material-to-a-3-d-object.md]: And here

**Review by Thraka — APPROVED** [review]
This all looks good but there are a few places that isn't changed. Were these missed some how?

**Comment by gewarren** [comment]
Thanks for the review, Andy.

**Pull request merged**